sklearnex/preview/ensemble/forest.py

```diff
@@ -1056,7 +1055,7 @@ def _onedal_fit(self, X, y, sample_weight=None, queue=None):
             'bootstrap': self.bootstrap,
             'oob_score': self.oob_score,
             'n_jobs': self.n_jobs,
-            'random_state': rs_,
+            'random_state': self.random_state,

```

Comment:
Same question about `check_random_state()` as in the other files.

Reply:
Same as above, now centralized in onedal/ensemble/forest.py